src/main/java/com/twentyonec/ItemsLogger/utils/Storage.java

```diff
@@ -0,0 +1,261 @@
+package com.twentyonec.ItemsLogger.utils;
+
+import java.sql.Connection;
+import java.sql.Date;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Time;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+
+import com.twentyonec.ItemsLogger.ItemPlayer;
+import com.twentyonec.ItemsLogger.ItemsLogger;
+import com.zaxxer.hikari.HikariConfig;
+import com.zaxxer.hikari.HikariDataSource;
+
+public class Storage {
+
+	private static Storage storage = null;
```

Comment:
Singletons are considered a terrible design pattern. I won't get into why now because it's a long, convoluted explanation but read up on dependency injection and why it's better.